### PR TITLE
Mobile menu with Javascript

### PR DESCRIPTION
**Pull request by DammyShittu**
## In this pull request,

Functionality was added to the portfolio through Javascript.

The mobile menu of the portfolio was worked on. Hence, when the portfolio is being checked on screens with their width being less than or equal to 800px, the navbar is hidden and a hamburger is displayed.

*Javascript was added to fulfill the following requirements*

- When the user clicks (or taps) the hamburger button, the mobile menu appears.
- When the user clicks (or taps) the close (X) button, the mobile menu disappears.
- When the user clicks (or taps) any of the mobile menu options, the mobile menu disappears.
- When the user clicks (or taps) any of the mobile menu options, a correct part of the page is displayed.

### CODE REVIEW

**Review by Shaher-11 — CHANGES_REQUESTED** [review]
#### Hello @DammyShittu,

### Great job so far!
You have been doing an incredible  job on the project so far:clap::clap:
The project is very well structured:smiley::smiley:
However, we still need to make a few changes to get the project approved:muscle:

 ## Required Changes ♻️

- [ ] Check the comment  below:point_down:

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Happy Coding!👏👏👏

**Comment by Davidosky007** [comment]
# Hi @lily-coder @DammyShittu, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
## Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Review by rpire — COMMENTED** [review]
Hello @DammyShittu and @lily-coder 

I like your work and everything you have done here, congratulations! 🤓

Cheers and Happy coding 🚀🙌

File: style.css [hunk 1]
```diff
@@ -672,6 +700,11 @@ footer {
     flex-direction: row;
     gap: 25px;
   }
+
+  .mobile-menu {
+    visibility: hidden;
+  }
+
```
Comment (rpire) [inline style.css]: Nice trick there, I like it! 👍

**Pull request merged**